docs/advanced/fetching-indexed-data.mdx

```diff
@@ -0,0 +1,156 @@
+# Fetching Indexed Data
```

Comment:
I think the error with the build is because you need to import the components:

```mdx
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';
import TOCInline from '@theme/TOCInline';

# Fetching Indexed Data

<TOCInline
  maxHeadingLevel={4}
  toc={toc}
/>

## Overview
```